tests/Feature/CelebrityTest.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Tests\Feature;
+
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
+use Tests\TestCase;
+use App\Models\User;
+
+class CelebrityTest extends TestCase
+{
+    /** @test  */
+    public function test_users_can_browse_celebrity()
+    {
+        $user = User::factory()->create();
+
+        $this->actingAs($user)->get('/celebrities')->assertOk();
```

Comment:
Formatting a bit here ;D The best way would be to the get and the assert ok in different lines. 

```
$this->actingAs($user)
       ->get...
        ->assertOk...
```

Also try to test the store and update and validation rules ;D